src/Symfony/Component/Translation/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.1
+---
+
+ * Parameters implementing `TranslatableInterface` are processed.
```

Comment:
we don't use final dots in changelog files

Reply:
done